app/server/utils/metrics/group.py

```diff
@@ -49,7 +69,11 @@ def build_query_group_by_with_join(self, query, query_object_model):
                     args = (grouped_query, self.custom_attribute_field_name)
                 else:
                     args = (grouped_query,)
-                return group_joining_strategies[query_object_model.__tablename__][self.group_object_model.__tablename__](*args)
+                if self.sender_or_recipient == 'recipient':
```

Comment:
` self.sender_or_recipient == 'recipient':` 
sorry coming into this code without context but this seems strange?

Reply:
Context for this change is [here](https://github.com/teamsempo/SempoBlockchain/pull/558), this is just extending the feature to groups from an existing feature for filters.